### PR TITLE
Basic graduation of k6/timers docs

### PR DESCRIPTION
**Pull request by mstoykov**
## What?

Add `k6/timers` that are just copy of `k6/experimental/timers`

More work is needed for this docs to be usable, but #1036 seems really relevant and is taking too long and might not fit in the v0.50.0 release cycle - so this the bare minimum so we can have the docs out.
## Checklist

- [x] I have used a meaningful title for the PR.
- [x] I have described the changes I've made in the "What?" section above.
- [x] I have performed a self-review of my changes.
- [x] I have run the `npm start` command locally and verified that the changes look good.

- [x] I have made my changes in the `docs/sources/next` folder of the documentation.
- [ ] I have reflected my changes in the `docs/sources/v{most_recent_release}` folder of the documentation.
- [x] I have reflected my changes in the relevant folders of the two previous k6 versions of the documentation (if still applicable to previous versions).

- [x] I have made my changes in the `docs/sources/next` folder of the documentation.

### CODE REVIEW

File: docs/sources/next/javascript-api/k6-timers/_index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: "k6/timers"
+description: "k6 timers API"
+weight: 13
+---
+
+# k6/timers
+
+Implement timers to work with k6's event loop. They mimic the commonly find in browser and other JavaScript runtimes.
```
Comment (heitortsergent) [inline docs/sources/next/javascript-api/k6-timers/_index.md:9]:
```suggestion
Implement timers to work with k6's event loop. They mimic the functionality found in browsers and other JavaScript runtimes.
```

File: docs/sources/next/javascript-api/k6-timers/_index.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+| Function                                       | Description                                                                                    |
+| :------------------------------------------ | :--------------------------------------------------------------------------------------------- |
+| [setTimeout](https://developer.mozilla.org/en-US/docs/Web/API/setTimeout)     | sets a function to be run after a given timeout  |
+| [clearTimeout](https://developer.mozilla.org/en-US/docs/Web/API/clearTimeout) | clears a previously set timeout with `setTimeout` |
+| [setInterval](https://developer.mozilla.org/en-US/docs/Web/API/setInterval)   | sets a function to be run on a given interval |
+| [clearInterval](https://developer.mozilla.org/en-US/docs/Web/API/setInterval) | clears a previously set interval with `setInterval` |
```
Comment (heitortsergent) [inline docs/sources/next/javascript-api/k6-timers/_index.md:16]:
```suggestion
| [clearInterval](https://developer.mozilla.org/en-US/docs/Web/API/setInterval) | Clears a previously set interval with `setInterval`. |
| [clearTimeout](https://developer.mozilla.org/en-US/docs/Web/API/clearTimeout) | Clears a previously set timeout with `setTimeout`. |
| [setInterval](https://developer.mozilla.org/en-US/docs/Web/API/setInterval)   | Sets a function to be run on a given interval. |
| [setTimeout](https://developer.mozilla.org/en-US/docs/Web/API/setTimeout)     | Sets a function to be run after a given timeout.  |
```

@mstoykov what do you think of ordering these alphabetically?

Comment (mstoykov) [reply docs/sources/next/javascript-api/k6-timers/_index.md:16]: I prefer to keep the order as they are grouped in how they are used which IMO is usually more useful.

**Review by heitortsergent — APPROVED** [review]

File: docs/sources/next/javascript-api/k6-timers/_index.md (hunk 2 above)
Comment (heitortsergent) [reply docs/sources/next/javascript-api/k6-timers/_index.md:16]: That's fair. :)

I updated my suggestion to keep the order, but just keep the suggestions for updating the descriptions.

**Pull request merged**